runtime/interpreter/value.go

```diff
@@ -2900,6 +2914,38 @@ func (v *ArrayValue) Slice(
 	)
 }
 
+func (v *ArrayValue) Reverse(
+	interpreter *Interpreter,
+	locationRange LocationRange,
+) Value {
+	count := v.Count()
+	idx := count - 1
```

Comment:
Using the full noun `index` is more intuitive 
```suggestion
	index := count - 1
```